omnisharp.json

```diff
@@ -1,5 +1,57 @@
 {
   "RoslynExtensionsOptions": {
-      "enableAnalyzersSupport": true
+    "enableAnalyzersSupport": true
+  },
+  "FormattingOptions": {
```

Comment:
This is where the formatting options actually live

Reply:
I don't believe that any of this would be picked up by Visual Studio, however there is a pretty modern way of having a shared config. I don't know if we're using a new enough version of omnisharp?

https://www.strathweb.com/2019/07/editorconfig-support-in-omnisharp-and-c-extension-vs-code/

Reply:
I think the omnisharp version is not a problem - seems like its coming from the vs code extension, which should keep it up to date afaict.
> OmniSharp server started.
    Path: C:\Users\JFM\.vscode\extensions\ms-vscode.csharp-1.21.5\.omnisharp\1.34.5\OmniSharp.exe
    PID: 25520

Also, looks like VS supports .editorconfig from 2017 onwards!
Defintiely seems worth using this in that case, thanks for bringing it up!